coverage/PENDING_REVIEW/intel/0488-COVERAGE-brw-fs-copy-propagation-691-brw-shader-565/cov-missing-return-value-function-never-called.amber

```diff
@@ -0,0 +1,230 @@
+#!amber
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A test for a coverage-gap found by the GraphicsFuzz project.
+
+# Short description: A fragment shader that covers specific BRW code paths
+
+# The test passes because the shader always writes red.
+
+SHADER vertex variant_vertex_shader PASSTHROUGH
+
+# variant_fragment_shader is derived from the following GLSL:
+# #version 320 es
+#
+# #define _int_0 _GLF_uniform_int_values[0]
+#
+# precision highp int;
+# precision highp float;
+#
+# // Contents of _GLF_uniform_int_values: 0
+# layout(set = 0, binding = 0) uniform buf0
+# {
+#     int _GLF_uniform_int_values[1];
+# };
+#
+# // Contents of one: 1
+# layout(set = 0, binding = 1) uniform buf1
+# {
+#     uint one;
+# };
+#
+# layout(location = 0) out vec4 _GLF_color;
+#
+# // Function is missing return value at the end,
+# // but this function is never called.
+# float func()
+# {
+#     switch(1)
+#     {
+#         case 0:
+#             return 1.0;
+#     }
+# }
```

Comment:
Are we sure we're OK with this? Is the generated SPIR-V definitely valid?

@paulthomson for info.

Reply:
SPIR-V looks fine to me. It returns undef at the end of the function:
```
         %17 = OpUndef %6
               OpReturnValue %17
```